src/document/operation/changeoperation.js

```diff
@@ -124,6 +97,35 @@ CKEDITOR.define( [ 'document/operation/operation', 'ckeditorerror' ], ( Operatio
 		getReversed() {
 			return new ChangeOperation( this.range, this.newAttr, this.oldAttr, this.baseVersion + 1 );
 		}
+
+		clone() {
+			return new ChangeOperation( this.range.clone(), this.oldAttr, this.newAttr, this.baseVersion );
+		}
+
+		/**
+		 * Checks whether this operation has conflicting attributes with given {@link document.operation.ChangeOperation}.
+		 * This happens when both operations changes an attribute with the same key and they either set different
+		 * values for this attribute or one of them removes it while the other one sets it.
+		 *
+		 * @param {document.operation.ChangeOperation} otherOperation Operation to check against.
+		 * @returns {boolean} True if operations have conflicting attributes.
+		 */
+		conflictsAttributesWith( otherOperation ) {
```

Comment:
Are you sure there won't be any other case where the user might like to compare two ChangeOperations? Well, on the other hand, we won't be encouraging users to create operations on their own but use `Transaction` API so you might be right. I'll move it.